jenkins: Drop RELEASE_NOTES parameter of kolibri-installer-android-upload

starnight opened the pull request:
We use kolibri-installer-android-upload job only to upload to internal testers. The real release notes are written in Google Play Console and only when promoting the testing build. For internal testers the notes aren't actually needed.

The parameter recentChangeList of androidApkUpload (Google Play Android Publisher Plugin) [1] is optional. It is passed as the RELEASE_NOTES. Drop RELEASE_NOTES to simplify parameters in Android upload Jenkins.

[1]: https://www.jenkins.io/doc/pipeline/steps/google-play-android-publisher/

https://phabricator.endlessm.com/T34808

dbnicholson reviewed (commented):
Unfortunately, I think the release notes are required by either the Jenkins plugin or the Google Play Publisher API. I think it would be better to just make the job default to an empty list (which I believe would correspond to no release notes) or a generic "Bug fixes and general improvements" message.

starnight committed:
jenkins: Add default value to RELEASE_NOTES of kolibri-installer-android-upload

We use kolibri-installer-android-upload job only to upload to internal
testers. The real release notes are written in Google Play Console and
only when promoting the testing build. For internal testers the notes
aren't actually needed.

Add an empty array as the default value to the parameter RELEASE_NOTES
of kolibri-installer-android-upload job.

https://phabricator.endlessm.com/T34808

starnight commented:
Updated as the new commit.
